spec/rubocop/cop/rspec/capybara/match_style_spec.rb

```diff
@@ -0,0 +1,62 @@
+# frozen_string_literal: true
+
+RSpec.describe RuboCop::Cop::RSpec::Capybara::MatchStyle, :config do
+  it 'registers an offense when using `assert_style`' do
+    expect_offense(<<~RUBY)
+      expect do
+        page.find(:css, '#first').assert_style(display: 'block')
+                                  ^^^^^^^^^^^^ Use `#good_method` instead of `#bad_method`.
+      end
```

Comment:
Does the `expect do .. end` make any sense?
I apologize for the confusion with asking if `.not_to raise_error` to make sense, I'm not really aware if it's a practice recommended by Capybara or not.

Reply:
Indeed it is. And I noticed in this comment that the offense message was still the default... 🙇🏻  Please check back as I have corrected it.